sass/px-context-browser-panel.scss

```diff
@@ -20,8 +20,8 @@
 
 // Component
 .pointer {
-  height: 16px;
-  width: 16px;
+  height: 0px;
```

Comment:
Why are these 0px/0px now?

Reply:
the pointer is all screwed up when it's at 16px, not sure why, but this fixes it